syft/messaging/protocol.py

```diff
@@ -80,8 +80,8 @@ def deploy(self, *workers):
 
         self._resolve_workers(workers)
 
-        for worker, plan in self.plans:
-            plan.send(worker)
+        for i, (worker, plan) in enumerate(self.plans):
```

Comment:
could we do this as list comprehension instead eg `self.plans = [(worker, plan.send(worker)) for (worker, plan) in self.plans]` (Im not 100% sure this works)